setup.py

```diff
@@ -1,7 +1,9 @@
+import os
 from setuptools import setup
 
 
-version = "0.11.1"
+VERSION_FILE = open(os.path.join("hyperdash", 'VERSION'))
+VERSION = VERSION_FILE.read().strip()
```

Comment:
Use consistent quote style.

Mind using `with` to ensure it's closed properly?
```python
with open(os.path.join('hyperdash', 'VERSION')) as f:
    VERSION = f.read().strip()
```